continuous-pipe.yml.erb

```diff
@@ -220,3 +248,17 @@ tasks:
               limits:
                 cpu: 1
                 memory: 1G
+
+  migration:
```

Comment:
it's worth knowing if for any reason the "default" pipeline is triggered, these tasks will all run in order of definition, so that's one reason why this was in the other way around rather than alphabetical order.